### ISSUE TITLE
`--exclude` flag could use better documentation

### ISSUE THREAD

**Issue by marrip** [editorconfig-checker/editorconfig-checker#259]
Hey,

I was a bit confused on how the `--exclude` flag works. I tried different approaches like defining it multiple times or separating the different regexes with spaces but nothing worked and the docs did not make any suggestion on how it is handled. I checked the code and realized that it should be `--exclude "regex1|regex2|regex3"`. I think it would be beneficial for users to have this somewhere explained to ease usage. Thank you 🙂

**Comment by generalmimon** [comment]
Yeah, currently only the last `-exclude` setting gets picked up and any previous setting is silently ignored (NB: no `\.rb$` in "Exclude Regexp"):

<pre><code>$ ec -verbose -exclude '\.rb$' -exclude '<ins><strong>\.gemspec$</strong></ins>'
Exclude Regexp: <ins><strong>\.gemspec$</strong></ins>|^\.yarn/|^yarn\.lock$|^package-lock\.json$|^composer\.lock$|^Cargo\.lock$|^Gemfile\.lock$|^\.pnp\.cjs$|^\.pnp\.js$|^\.pnp\.loader\.mjs$|\.snap$|\.otf$|\.woff$|\.woff2$|\.eot$|\.ttf$|\.gif$|\.png$|\.jpg$|\.jpeg$|\.webp$|\.avif$|\.mp4$|\.wmv$|\.svg$|\.ico$|\.bak$|\.bin$|\.pdf$|\.zip$|\.gz$|\.tar$|\.7z$|\.bz2$|\.log$|\.patch$|\.css\.map$|\.js\.map$|min\.css$|min\.js$
</code></pre>

Ironically, this project's own `Makefile` doesn't account for this either:

https://github.com/editorconfig-checker/editorconfig-checker/blob/585ef52545d466ed68ac6147f0c7b4e964e66820/Makefile#L52-L56

It would be great if multiple `-exclude`s were all picked up and merged into the regexp as alternatives (like the `Exclude` array in JSON configuration), as many of us would intuitively expect. An inferior alternative (but still better than the current silent "last wins" overwrite) would be to detect multiple `-exclude` settings and inform the user that only the last one was applied - but I guess that the proper fix of just adding support for multiple `-exclude`s would actually be easier to implement than this ;)

**Comment by mstruebing** [comment]
Anyone willing to create a PR for this?

**Comment by ngotchac** [comment]
I too found the `exclude` flag not so easy to use. I expected to be able to pass multiple `--exclude`, each being a glob rather than a regex. Wouldn't that make more sense?
